tools/src/postgen-bytetypes.ts

```diff
@@ -0,0 +1,86 @@
+const pascalcase = require('pascalcase');
```

Comment:
I see this is already a dependency, but I thought I'd point out it's 16 lines of code with a single maintainer and no updates in 2 years.